lib/beacon_api/controllers/v1/beacon_controller.ex

```diff
@@ -3,13 +3,14 @@ defmodule BeaconApi.V1.BeaconController do
   alias BeaconApi.ErrorController
 
   alias LambdaEthereumConsensus.ForkChoice
+  alias LambdaEthereumConsensus.Store.Blocks
   alias LambdaEthereumConsensus.Store.BlockStore
   use BeaconApi, :controller
 
   plug(OpenApiSpex.Plug.CastAndValidate, json_render_error_v2: true)
 
   @doc """
-  action is an atom that correspond to the controller action's function atoms declared on `BeaconApi.Router` 
+  action is an atom that correspond to the controller action's function atoms declared on `BeaconApi.Router`
```

Comment:
would remove this @doc

Reply:
I'll leave it as a comment. It's important to document this, since the function creates (a bounded anount of) new atoms at runtime.